library/kani_macros/src/sysroot/contracts/shared.rs

```diff
@@ -50,7 +50,7 @@ impl<'a> ContractConditionsHandler<'a> {
     pub fn emit_common_header(&mut self) {
         if self.function_state.emit_tag_attr() {
             self.output.extend(quote!(
-                #[allow(dead_code, unused_variables)]
+                #[allow(dead_code, unused_variables, unused_mut)]
```

Comment:
Could you add or modify an existing test case to confirm this fix? What you can do is to update some existing `.expected` file to make sure there are no more warnings of this kind.

Reply:
Changes to the .expected file didn't seem to do it; I tried instead binding x and y as mut in the original gcd test and adding #![deny(warnings)]. I've confirmed locally that #![deny(warnings)] will cause the regression test to fail unless you add unused_mut.